toxicity/src/index.ts

```diff
@@ -30,12 +30,17 @@ declare interface ModelInputs extends tf.NamedTensorMap {
  *
  * @param threshold A prediction is considered valid only if its confidence
  * exceeds the threshold. Defaults to 0.85.
- * @param toxicityLabels An array of strings indicating which types of toxicity
+ * @param toxicityLabels  A string or an array of strings indicating which types of toxicity
```

Comment:
This fails lint due to over 80 columns

Step #2 - "lint": ERROR: (max-line-length) /workspace/toxicity/src/index.ts[33, 1]: Exceeds maximum line length of 80
Step #2 - "lint":